flink-connector-mysql-cdc/src/test/java/com/ververica/cdc/connectors/mysql/MySqlTestUtils.java

```diff
@@ -59,6 +59,7 @@ public static MySqlSource.Builder<SourceRecord> basicSourceBuilder(
                 .username(database.getUsername())
                 .password(database.getPassword())
                 .deserializer(new ForwardDeserializeSchema())
+                .serverTimeZone("UTC")
```

Comment:
minor: only `testValidateTimezone` need set the parameter, but we set it for all legacy tests.